### PR TITLE
Loading tips update, by loop

### PR DESCRIPTION
**Pull request by lupuleasa-core**
let me know if everything looks ok

### CODE REVIEW

**Comment by lupuleasa-core** [comment]
@AtobaAzul let me know if this is merge ready or if you need other modifications
I made most of the character specific ones to be survivor quote format

File: init/init_loadingtips.lua [hunk 1]
```diff
@@ -1,8 +1,7 @@
 function setup_custom_loading_tips()
     local tips = {
-        ["TOPHAT"] = "\"I had left the top hat behind with my old act, but it's still good for a magic trick or two.\" -M",
         ["AMALGAMS"] = "\"Whoever designed these clockwork thingamawatzits should have installed a surge protector!\" -W",
-        ["RNES"] = "\"I feel like there's something watching us at night...\" - W",
+        ["RNES"] = "\"I feel like there's something watching us at night... Huh?!... What was THAT?! What IS that?!\" - W",
```
Comment (AtobaAzul) [inline init/init_loadingtips.lua:4]: Not a big fan of this one.

File: init/init_loadingtips.lua [hunk 2]
```diff
@@ -11,61 +10,75 @@ function setup_custom_loading_tips()
         ["RATS_FOODSCORE"] = "\"Our rations appear to be attracting unwanted attention. I should get rid of our stale food...\" - W",
         ["RATS_ITEMSCORE"] = "\"The vermin have noticed the mess around camp, I really should do a bit of Spring cleaning...\" - W",
         ["RATS_BURROWBONUS"] = "\"I've spotted a rat den where there wasn't one before, I think they are multiplying, and fast!\" - W",
-        ["SNOWPILES"] = "\"The snow is accumulating here rather fast. I should dig it soon before it covers everything.\" - W",
+        ["SNOWPILES"] = "\"The snow is accumulating here rather fast. We should dig it soon before it covers everything. Wait, did you SEE that pile move??\" - W",
```
Comment (AtobaAzul) [inline init/init_loadingtips.lua:13]: The 'I' to 'We' change is good, but I don't like the added text.

File: init/init_loadingtips.lua [hunk 3]
```diff
@@ -11,61 +10,75 @@ function setup_custom_loading_tips()
...
         ["NOCOLLISION"] = "Most exploitable collisions have been removed. This includes signs, statues, giant crops, shell clusters, and more.",
         ["MAXHPLOSS"] = "Freezing, overheating, starving and more can reduce max health.",
         ["WEATHER"] = "Keep an eye out on each season. Every season has something new to encounter.",
-        ["WARLY_BUTCHER"] = "As Warly, killing critters in your inventory grants double drops.",
         ["MAXHEALTHHEALING"] = "Warly's Salt Spice can restore lost max health.",
         ["SLEEPING"] = "Sleeping has been considerably improved. Stats are gained faster and can health lost max health up to a certain threshold.",
-        ["WALTER_WOBY"] = "\"I thought Woby some new tricks! Look! She can grab what I command her and bark!\" - W",
-        ["ALPHAGOAT"] = "\"That's a mean lookin' goat. I bet it'd make some fine dinin'\" - W",
+        ["ALPHAGOAT"] = "\"That's a mean lookin' goat. I bet it'd make some fine dinin'! Ouch ouch ouch! Run! He's angry!\" - W",
```
Comment (AtobaAzul) [inline init/init_loadingtips.lua:25]: Loading tips are written more like journaling rather than actual quotes.

File: init/init_loadingtips.lua [hunk 4]
```diff
@@ -11,61 +10,75 @@ function setup_custom_loading_tips()
...
+        ["FAVORITE_FOOD"] = "A survivor's favorite food is a great thing to keep in mind when one wishes for some peace of mind.",
+        ["LAZY_DESERTER"] = "The lazy deserter is now also a lazy collector! Just make sure to to mind your sanity.",
+        ["MONSTERMEAT_CROCKPOT"] = "\"Monster meat is SO horrendous to cook with. But a true chef knows to mix it up with some healthy meat if the situation calls for it.\" - W",
+        ["THERMAL_STONE"] = "Thermal stones now become much better with proper clothing, and much worse without.",
+        ["LIFE_AMULET"] = "Ghosts can no longer haunt Life Amulets to revive. Unless those amulets are of the rare and ancient variety.",
+        ["PEARL_SHOP"] = "Pearl has expanded shop, and has a few more bits and baubles to sell.",
+
+        --character specific
+        ["WARLY_BUTCHER"] = "\"Warly is a great friend to have. I have been capturing these creatures alive lately. He has a way with a knife that I cannot match.\" - W",
+        ["WINONA_ELECTRICAL"] = "Winona expanded her electrical arsenal quite a bit! You should see her pack up her gizmos on the go. She even made my HAT and lantern batteries included!",
```
Comment (AtobaAzul) [inline init/init_loadingtips.lua:71]: Last bit is a bit redundant considering the next tip.

**Review by AtobaAzul — COMMENTED** [review]
Overral, very good! I'll have to do some tweaks myself, though.

**Pull request merged**